go/run_lemon-mint5.sh

```
@@ -0,0 +1,4 @@
+cd go
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)